cognite/neat/rules/models/_rules/information_rules.py

```diff
@@ -499,3 +492,27 @@ def _to_space(cls, prefix: str) -> str:
         if prefix[-1] == "_":
             prefix = f"{prefix[:-1]}1"
         return prefix
+
+    @classmethod
+    def _get_container(cls, prop: InformationProperty) -> tuple[ContainerEntity, str]:
+        if isinstance(prop.reference, ReferenceEntity):
+            return (
+                ContainerEntity(prefix=prop.reference.prefix, suffix=prop.reference.suffix),
+                prop.reference.property_ or prop.property_,
+            )
+        else:
+            return ContainerEntity(prefix=prop.class_.prefix, suffix=prop.class_.suffix), prop.property_
+
+    @classmethod
+    def _get_view_implements(cls, cls_: InformationClass) -> list[ViewEntity]:
+        if isinstance(cls_.reference, ReferenceEntity):
```

Comment:
They can use either parent class or reference, not both. Important to stress them out this in tutorial, because we need to assume that ppl will start mixing things up.